deleteme.txt

```diff
@@ -0,0 +1 @@
+blah
```

Comment:
I think this is the only thing left to do, right? Once you remove this, is this ready to be merged? Also check the test failures (slow to respond right now, will focus more later today)